utils/mkhtml.py

```diff
@@ -293,7 +293,7 @@ def get_addon_path(pgm):
 
     :param pgm str: pgm
 
-    :return tuple: (True, path) if pgm is addon else (None, None)
+    return: True if pgm is addon else None
```

Comment:
Perhaps returning bool always would be more clear, True or False, not True or None.